### PR TITLE
blog: Add monthly updates post

### PR DESCRIPTION
**Pull request by zapbot**
<details>
<summary> Screenshot </summary>

image

</details>

### CODE REVIEW

File: site/content/blog/2024-11-01-zap-updates-october-2024/index.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
+---
+title: "ZAP Updates - October 2024"
+summary: >
+  TODO
+images:
+- https://www.zaproxy.org/blog/2024-11-01-zap-updates-november-2024/images/zapbot-monthly-updates.png
```
Comment (ricekot) [inline site/content/blog/2024-11-01-zap-updates-october-2024/index.md:6]:
```suggestion
- https://www.zaproxy.org/blog/2024-11-01-zap-updates-october-2024/images/zapbot-monthly-updates.png
```

File: site/content/blog/2024-11-01-zap-updates-october-2024/index.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+A lot of exciting updates across add-ons are in the pipeline.
+- New GraphQL fingerprinting checks were added for a few engines
+- A new "ASCIfy" encoder was added to strip all non-ASCII characters from selected strings
+- Automation framework enhancements like simplified YAML plans, and some cool new automation jobs
+
+## Join the Discussion
+
+### ZAP Scan Policies
+
+We are considering including a default set of Active Scan Policies with all ZAP releases.
```
Comment (kingthorin) [inline site/content/blog/2024-11-01-zap-updates-october-2024/index.md:56]: Should this be worded differently, the plan is an add-on right?

File: site/content/blog/2024-11-01-zap-updates-october-2024/index.md [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+These will be define the rules and associated strengths and thresholds that we recommend using in different situations.
```
Comment (kingthorin) [inline site/content/blog/2024-11-01-zap-updates-october-2024/index.md:57]: Drop "be"

File: site/content/blog/2024-11-01-zap-updates-october-2024/index.md [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+For example, sets of policies for different types of ZAP users, e.g. Developers or QA testers, or policies that include rules defined by popular standards like the OWASP Top 10 or Google CASA.
+
+Read the full discussion and share your thoughts on the [ZAP User Group](https://groups.google.com/g/zaproxy-users/c/bEBOGB48v-I/m/3r5TRPdVAwAJ).
+
+### Backlog Issues
+
+We've added a new `backlog` tag for ZAP issues in GitHub.
```
Comment (kingthorin) [inline site/content/blog/2024-11-01-zap-updates-october-2024/index.md:64]: in ➡️ on

File: site/content/blog/2024-11-01-zap-updates-october-2024/index.md (hunk 2 above)
Comment (ricekot) [reply site/content/blog/2024-11-01-zap-updates-october-2024/index.md:56]: I copied that from https://groups.google.com/g/zaproxy-users/c/bEBOGB48v-I/m/3r5TRPdVAwAJ, but happy to change it. Does the following read better?

> We are considering including a default set of Active Scan Policies with ZAP as part of a new add-on.

**Review by psiinon — APPROVED** [review]
LGTM!

File: site/content/blog/2024-11-01-zap-updates-october-2024/index.md (hunk 2 above)
Comment (kingthorin) [reply site/content/blog/2024-11-01-zap-updates-october-2024/index.md]: I think so. @psiinon ? @thc202 ?

Comment (psiinon) [reply site/content/blog/2024-11-01-zap-updates-october-2024/index.md]: Yeah, that looks better.

Comment (thc202) [reply site/content/blog/2024-11-01-zap-updates-october-2024/index.md]: I'm good with that.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

File: site/content/blog/2024-11-01-zap-updates-october-2024/index.md (hunk 2 above)
Comment (ricekot) [reply site/content/blog/2024-11-01-zap-updates-october-2024/index.md]: Already pushed that :)

**Pull request merged**